src/azure-cli/azure/cli/command_modules/keyvault/commands.py

```diff
@@ -64,6 +77,27 @@ def load_command_table(self, _):
         g.custom_command('remove', 'remove_network_rule')
         g.custom_command('list', 'list_network_rules')
 
+    with self.command_group('keyvault private-endpoint',
+                            kv_private_endpoint_connections_sdk,
+                            min_api='2018-02-14',
+                            client_factory=keyvault_client_private_endpoint_connections_factory,
+                            is_preview=True) as g:
+        g.custom_command('approve', 'approve_private_endpoint_connection',
+                         validator=validate_private_endpoint_connection_id)
+        g.custom_command('reject', 'reject_private_endpoint_connection',
+                         validator=validate_private_endpoint_connection_id)
```

Comment:
`approve` and `reject` are not common CLI commands. An alternative ways may be putting them in `keyvault private-endpoint update --approve true`. Has this been confirmed with PowerShell team @isra-fel or Key Vault PM?

Reply:
@jiasli Yes, this is confirmed with Key Vault team, will forward you the email later.

Reply:
here. I think this design is good.